examples/ReactExample/components/login.js

```diff
@@ -0,0 +1,102 @@
+'use strict';
+
+import React from 'react';
+import { AsyncStorage, Text, TextInput, View } from 'react-native';
+import Realm from 'realm';
+import styles from './styles';
+import TodoApp from './todo-app';
+
+
+export default class LoginScreen extends React.Component {
+
+    constructor (props) {
+        super(props);
+        this.state = {login: 'user', password: '...'};
+        this._submit = this.submit.bind(this);
+    }
+
+    componentDidMount () {
+        AsyncStorage.getItem('creds').then( value_str => {
+            console.log('CREDS', value_str);
+            if (value_str) {
+                const value = JSON.parse(value_str);
+                this.setState({
+                    login: value.login||'user',
+                    password: value.password||'password'
+                });
+            }
+        } ).done();
+    }
+
+    submit () {
+        console.log('LOGIN')
+        Realm.Sync.User.login(
+            "http://192.168.157.64:9080/",
```

Comment:
I suppose we should extract this to somewhere. Where is it stored in the other examples?